code-challenges.test.js

```diff
@@ -20,35 +20,100 @@ var colors1 = ["purple", "blue", "green", "yellow", "pink"]
 var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
+describe('shuffleArray', () => {
+  it('should return a randomized instance of a given array', () => {
+    const randomized1 = shuffleArray(colors1);
+    const randomized2 = shuffleArray(colors2);
+
```

Comment:
It seems you are calling your function here and holding the return value in your variables that are then passed to your test. I would be cautious of this strategy as your test and function become intertwined.